weave/trace_server/migrations/005_add_cost.up.sql

```diff
@@ -0,0 +1,39 @@
+CREATE TABLE llm_token_prices (
+    /*
+    `pricing_level`: The level at which the token pricing applies. It can be 'default' for global pricing,
+    'project' for project-specific pricing, or 'org' for organization-specific pricing.
+    */
+    pricing_level String,
+
+    /*
+    `pricing_level_id`: The pricing level identifier for the token pricing. It can be 'default' or any other string(e.g. org id or project id). This allow with type to uniquely identify where the pricing applies.
+    */
+    pricing_level_id String,
```

Comment:
what is the value here if `pricing_level=default`?

Reply:
"default" as well